examples/LAMMPS_OPENFOAM/minimal_CFD.cpp

```diff
@@ -0,0 +1,40 @@
+#include "mpi.h"
+#include <iostream>
+
+#include "cpl.h"
+#include "CPL_ndArray.h"
+
+using namespace std;
+
+int main() {
+
+    bool flag;
+    int CFD_realm = 1;
+    MPI_Comm CFD_COMM, CART_COMM;
+    CPL::ndArray<double> send_array, recv_array;
+
+    MPI_Init(NULL, NULL); 
+    CPL::init(CFD_realm, CFD_COMM);
+
+    int npxyz[3] = {1, 1, 1}; int periods[3] = {1, 1, 1};
+    MPI_Cart_create(CFD_COMM, 3, npxyz, periods, 1, &CART_COMM);
+
+    //MPI_Cart_get(icomm_grid, 3, npxyz_cfd, cart_periods, cart_coords);
+
+    double xyzL[3] = {8*2.099495, 8*5.668637, 8*2.099495}; double xyz_orig[3] = {0.0, 0.0, 0.0};
+    int ncxyz[3] = {8, 8, 8};
+    CPL::setup_cfd(CART_COMM, xyzL, xyz_orig, ncxyz);
+    CPL::get_arrays(&recv_array, 4, &send_array, 1);
```

Comment:
4 here  works with the current minimal MD; but 3 also works fine if changed on the MD side too